translations/ui-data-import/en_US.json

```diff
@@ -108,10 +108,11 @@
     "communicationProblem": "Communication problem with server. Please try again",
     "settings.jobProfiles.action.success": "The job profile \"{name}\" was successfully {action}",
     "settings.jobProfiles.action.error": "The job profile \"{name}\" was not {action}",
+    "modal.jobProfile.delete.header": "Delete \"{name}\" job profile?",
+    "modal.jobProfile.delete.message": "Delete job profile?",
```

Comment:
Usually, we do not add changes to that file and deal only with `en.json`. Please changes from that file.